addons/project_sale_expense/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Project - Sale - Expense',
+    'version': '1.0',
+    'description': 'Bridge between sale_project and sale_expense modules',
+    'license': 'LGPL-3',
+    'category': 'Hidden',
+    'depends': ['sale_project', 'sale_expense'],
```

Comment:
```suggestion
    'depends': ['sale_project', 'sale_expense', 'project_hr_expense'],
```
the same reason `sale_purchase_stock` has three dependencies.

Reply:
Indeed, thanks. 🙂

Reply:
Indeed, thanks. 🙂